index.html

```diff
@@ -48,7 +41,7 @@ <h1 class="intro">Welcome to YAOTIAN TONG’s CSD120 Website</h1>
        </section>
         
         
-        <section class="bio">
+        <section class="favourites">
```

Comment:
The class names here should reflect the meaning of the element's contents.  This is not the bio section.  I gave the section a more appropriate name.